articles/spotify-python-part-1/spotipy_spotify_python.md

```diff
@@ -0,0 +1,131 @@
+![img](py_img.jpg)
+
+# Spotify API with Spotipy Part 1
+
+## Introduction 
+
+Spotify is a wonderful music streaming app with powerful API functionality. The API is both beginner-friendly and offers a wide range of endpoints. This makes Spotify API a great way to get started for beginner programmers who are interested in getting involved in web APIs and even data science or advanced programmers who want more control over their playlists. Spotipy is a popular wrapper for the Spotify API. It enables and supports all the features of the Spotify API but allows them to be designed and run in Python. This allows for scalability and also will come in handy when we start dealing with large amounts of data.
+
+## Goals 
+
+This is part one of a three-part tutorial series on Spotipy and its applications. By the end, you should be able to design and implement algorithms for creating unique playlists, search for recommended music, and analyze metadata about your music selection.
+
+To achieve these lofty goals will require a strong foundation. That's the goal of part 1 of this series. By the end, you should understand how to set up a spotipy project, the authentification process, and finishing it off. We'll start using some of the spotipy endpoints.
+
+## Prerequisites
+
+1. You need a Spotify premium account
+
+2. I try to make my tutorials as simple as possible, so even if you are a beginner programmer, you can follow along and achieve something you're proud of. That being said, you should have some basic knowledge of Python before starting. If you know how to use the [enumerate()](https://docs.python.org/3/library/functions.html#enumerate) function, how to [import modules](https://docs.python.org/3/tutorial/modules.html), and how to use pip3, you should be able to complete this part of the tutorial. If you are not confident in these skills, yet it is okay! Click on the links and when you feel confident, return to the article.
+
+3. You should understand how to use the terminal
+
+
+## Step 1: Set up a Developer account
+
+
+The first step is to create a [Spotify developer account](https://developer.spotify.com/dashboard/). Click on the link, sign in to your Spotify account, and follow the instructions to create a developer account.
+For the redirect URL, it is highly recommended to use http://localhost:9000 or the localhost port of your choosing.
+
+Trust me, I have tried using a different website, and it usually creates bugs. Localhost is the best option for the redirect link.
```

Comment:
Don't use trust me, I speak with experience, etc. It comes across as way too informal.